src/devtools/client/debugger/src/components/Editor/Gutter.tsx

```diff
@@ -4,15 +4,10 @@ import LineHitCounts from "./LineHitCounts";
 import ToggleWidgetButton from "./ToggleWidgetButton";
 
 export default function Gutter({ editor }: any) {
-  const [isLineHitCountsCollapsed, setIsLineHitCountsCollapsed] = useState(false);
```

Comment:
I moved the state into redux so it would be easier to synchronize with prefs. Most of the changes below are there because of the move to redux.